src/GeneralStateTestsFiller/stCreate2/CREATE2_HighNonceFiller.yml

```diff
@@ -0,0 +1,56 @@
+# The test calls CREATE2 from an account with max allowed nonce.
+CREATE2_HighNonce:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentBaseFee: '10'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 1000000000
+      code: ''
+      nonce: 0
+      storage: {}
+    b94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 0
+      code: |
+        :yul
+        {
+          // initcode: { return(0, 1) }
+          mstore(0, 0x60016000f3000000000000000000000000000000000000000000000000000000)
+          sstore(0, create2(0, 0, 5, 0))
+          sstore(1, 1)
+        }
+      nonce: '0xffffffffffffffff'
```

Comment:
Could we also add here a positive test case CREATE2_HighNonceMinus1, where nonce is '0xfffffffffffffffe' and contract '0x77dd5d2a2b742ca01ee2cfff306445e3741ef744' is actually created?

Reply:
Added.